src/mpid/ch4/shm/xpmem/xpmem_pre.h

```diff
@@ -14,6 +14,41 @@
 #include <mpi.h>
 #include <xpmem.h>
 
+#define RLOW_LOCATE_MIN_RIGHT(node, rlow) (rlow >= node->min)
+#define RHIGH_LOCATE_MAX_LEFT(node, rhigh) (rhigh <= node->max)
+#define RLOW_LOCATE_LOW_LEFT(node, rlow) (rlow < node->low)
+#define RHIGH_LOCATE_HIGH_LEFT(node, rhigh) (rhigh <= node->high)
+
+#define MPIDI_XPMEM_AVL_TREE_BUFFER_NUM 64
+#define MPIDI_XPMEM_AVL_TREE_BUFFER_NODE_NUM 4096
+
```

Comment:
Please add short description about these threshold. Ideally, it is better to define CVAR environment variables (see the pt2p2 threshold CVAR)